src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -46,6 +46,22 @@ public void Dispose()
             var server = new TcpListener(endPoint);
             server.Start();
 
+            _clientCertificateRemovedByFilter = false;
+
+            if (PlatformDetection.IsWindows7 && 
+                !useClientSelectionCallback && 
+                !Capability.IsTrustedRootCertificateInstalled())
+            {
+                // https://technet.microsoft.com/en-us/library/hh831771.aspx#BKMK_Changes2012R2
+                // Starting with Windows 8, the "Management of trusted issuers for client authentication" has changed:
+                // The behavior to send the Trusted Issuer List by default is off.
+                //
+                // In Windows 7 the DN list is sent within the Server Hello. If the client selection callback isn't 
+                // used and the client certificate's issuer is not in the server's Trusted Root Authorities the client,
+                // not send the certificate.
```

Comment:
>A client certificate will not be sent (filtered-out) if:
1. The client selection callback is not used.
2. The server did not send the client certificate issuer within the Trusted Issuer List. 

I think you should clarify that this logic is only true if the server sends a Trusted Issuer List.

If there is no list sent by the server, then the client can/will sent a cert from the client to server. It's just that the client doesn't have a list to filter the eligible client certificate with.  So, the eligibility list is larger based on the set of client certificates in the SslStream API call.